src/messages/data.ts

```diff
@@ -0,0 +1,46 @@
+import { base64url } from 'multiformats/bases/base64';
+import { CID } from 'multiformats/cid';
+import { importer } from 'ipfs-unixfs-importer';
+
+export class Data {
+  data: string | number | boolean | Uint8Array | object;
```

Comment:
Do we really want this much optionality after constructions? The way I see it, unless we are able to deserialize them back to their original form, I think it is fine just to have data being `Unit8Array`, and we just call `toBytes()` at construction.

Reply:
nixed the `Data` class. it's now within `utils`.

Reply:
Going to de-dupe with the next comment below.